src/xnnpack/microfnptr.h

```diff
@@ -73,6 +73,20 @@ typedef void (*xnn_dqgemm_ukernel_fn)(
     const void* params,
     const struct xnn_qd8_quantization_params* quantization_params);
 
+typedef void (*xnn_dqgemm_bl_ukernel_fn)(
```

Comment:
can we drop this declaration? We can defer introducing this until we introduce operator level APIs.